Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -181,9 +183,13 @@ Common::Debug::Threads PPCDebugInterface::GetThreads() const
   if (!active_thread->IsValid())
     return threads;
 
-  const auto insert_threads = [&threads](u32 addr, auto get_next_addr) {
+  std::vector<u32> visited_addr{{active_thread->GetAddress()}};
```

Comment:
Would a set or another container be a better choice?

Reply:
`vector<>` tends to perform really well for small sizes on modern systems thanks to cache locality, and I can't imagine the amount of addresses here reaching even three digits. So I can't imagine this mattering much.